lib/YaST/Kdump/KdumpController.pm

```diff
@@ -0,0 +1,69 @@
+# SUSE's openQA tests
```

Comment:
would you like to be the first to enable libyui-rest-api without Controllers in this task, we don't need them in most of the cases as I mentioned since a lot of time ago.
Basically we need to:
- Move Controller's getters to corresponding Page
- Make a chain call with those getters for each action in the Page, `$self->getter_method()->page_action_method()`

Otherwise we can postpone this for new ticket.